run_iwyu_tests.py

```diff
@@ -69,7 +69,8 @@ def setUp(self):
                                     self.CheckAlsoExtension('-fnreturn.h'),
                                     self.CheckAlsoExtension('-typedefs.h'),
                                     self.CheckAlsoExtension('-d2.h')],
-      'keep_mapping.cc': [self.CheckAlsoExtension('-public.h'), 
+      'keep_includes.c' : [ '--keep=./tests/c/keep_b.h' ],
```

Comment:
I think you can get fancy here and use a glob -- how about `'--keep="*keep_b.h"'`?

Reply:
Also, in line with convention, the header should be called `keep_includes-d1.h` -- that makes it clear that it belongs to the `keep_includes` test, and the `d` indicates it's a direct include.

Reply:
Oh, and no space before colon, and no spaces inside list brackets.